requirements/default.txt

```diff
@@ -150,8 +150,7 @@ django-pipeline==1.6.14 \
 django-session-csrf==0.7.1 \
     --hash=sha256:e17177e6e2e6518ec7ce6693ad10a5c747f8571d09f4cfa9082599334421605d \
     --hash=sha256:ff8c10e30d312c77fc6a6db7710e22b9383e28c03b7fe958876ca96f39aa6cf2
-# the 'linux2' is for TravisCI
-pyinotify==0.9.6 ; sys_platform == 'linux' or sys_platform == 'linux2' \
+pyinotify==0.9.6 ; sys_platform == 'linux' \
```

Comment:
Circle failed--we need to re-add "linux2" here. I don't know why the comment was like that, but probably my fault for not updating it for Circle.